Learning-Labs-101/2-theymlfile.md

```diff
@@ -0,0 +1,268 @@
+# 🛠 Working With Github Learning Labs: The YAML File
```

Comment:
can we add this into the repo_logistics folder?

Reply:
@lazyplatypus should all learning lab content go in repo-logistics then?

Reply:
so we can perhaps do something like

repo_logistics/
- learninglab.md
- readme.md
- example_structure/ ....?

Reply:
sounds good; there are actually 3 learning lab files so it would be
repo_logistics/
* learninglab101/
* readme.md
* example_structure/

Reply:
instead of learninglab101 can we do learning_lab

Reply:
@lazyplatypus there is actually a separate PR for learning lab content, so I will remove the learning lab files from this PR and change the directory structure in the other.